similarity/metrics.py

```diff
@@ -0,0 +1,290 @@
+import numpy as np
+
+import db.data
+import db.stats
+from operations import BaseMetric
+
+NORMALIZATION_SAMPLE_SIZE = 10000
+
+
+class LowLevelMetric(BaseMetric):
+    path = ''
+    indices = None
+
+    def get_data(self, id):
```

Comment:
sorry that I didn't reply to this - one easy way would be to just mock the call to db.data, as long as that method also has a test, then this would be enough for the test